student-test/assignment1/Test_validate_username.c

```diff
@@ -18,5 +18,9 @@ void test_validate_my_username()
      * TODO: Replace the line below with your code here as described above to verify your /conf/username.txt 
      * config file and my_username() functions are setup properly
      */
-    TEST_ASSERT_TRUE_MESSAGE(false,"AESD students, please fix me!");
+    const char *str1 =  my_username();
+    const char *str2 = malloc_username_from_conf_file();
+
+    //TEST_ASSERT_TRUE_MESSAGE(false,"AESD students, please fix me!");
+    TEST_ASSERT_EQUAL_STRING_MESSAGE(str1,str2,"Pass");
```

Comment:
@dhirajbennadi what happens to `str2` here?  See caller requirements for `malloc_username_from_conf_file` in the comments on this function.